test/test_scanner.py

```diff
@@ -53,7 +53,8 @@ def setup_class(cls):
 
     @classmethod
     def teardown_class(cls):
-        cls.cve_scanner.__exit__()
```

Comment:
Why are we not calling `__exit__`? Shouldn't it call `.close()` for cursor and connection?